kotest-assertions/kotest-assertions-shared/src/jvmMain/kotlin/io/kotest/assertions/eq/isOrderedSet.kt

```diff
@@ -1,11 +1,11 @@
 package io.kotest.assertions.eq
 
-import java.util.SortedSet
+import java.util.*
 
 /**
  * JVM-specific: Allows TreeSet to be considered ordered for iterable comparisons
  */
 actual fun isOrderedSet(item: Iterable<*>) =
    item is LinkedHashSet ||
-      item is SortedSet ||
+      item is SequencedSet ||
```

Comment:
do we need to add/change any tests to validate this code change?

Reply:
> do we need to add/change any tests to validate this code change?

well... for some java `linkedHashSet` I think so?